index.html

```diff
@@ -926,13 +954,18 @@ <h2>Client-Managed Version Creation</h2>
          <h2>Replacing Contents from Mementos</h2>
           <blockquote class="informative">
             Non-normative note:
-            Using the ingest-by-reference mechanism, one can replace the contents of an <a>LDPRv</a> with that of
-            an <a>LDPRm</a> by providing it's URL as the <code>URL</code> parameter in a <code>Content-Type:
-            message/external-body</code> header.  For example, given an <a>LDPRm</a> with URL <code>
-            http://example.org/some/memento</code>, the full header would be <br/>
-            <code>Content-Type: message/external-body; access-type=URL; expiration=1;<br/>
-            &nbsp;&nbsp;&nbsp;&nbsp;URL="http://example.org/some/memento"</code>
+            Using the ingest-by-reference mechanism provided by the <code>external-content-handling=copy</code>
+            preference described in <a href="#external-content"></a>, servers may support replacement of the contents
+            of an <a>LDPRv</a> with that of an <a>LDPRm</a> by providing the <a>LDPRm</a>'s location as the target
+            of a link indicating an external content request. For example, given an <a>LDPRm</a> with URL
+            <code>http://example.org/some/memento</code>, the appropriate headers would be:
           </blockquote>
+          <div class="example">
+<pre>Link: &lt;http://example.org/some/memento&gt;;
+      rel="http://fedora.info/definitions/fcrepo#ExternalContent";
+      handling="copy"
```

Comment:
This is a good, informative section.  If the server supports the `copy` handling, then the above example is still  _may or may not_ work because of the absence of `type=`?  i.e servers are within their right to reject the request due to the absence of a type, might assign a different (default) mime type to the `LRPRv`,  but might also do the Right Thing implicitly.

Reply:
Yes, better to include a `type="image/tiff"` -- I suspect that implementations will be written to do the right thing but since we aren't requiring it, it seems better to be explicit